blivet/devices/luks.py

```diff
@@ -140,3 +140,34 @@ def populate_ksdata(self, data):
         self.slave.populate_ksdata(data)
         data.encrypted = True
         super(LUKSDevice, self).populate_ksdata(data)
+
+
+class IntegrityDevice(DMIntegrityDevice):
```

Comment:
So it is possible (expected?) to have dm-integrity devices that are not associated with LUKS2 devices?

Reply:
I found out later that this is indeed possible.